co-log/src/Colog/Message.hs

```diff
@@ -166,14 +174,14 @@ type FieldMap (m :: Type -> Type) = TypeRepMap (MessageField m)
 'UTCTime'. Basically, the following mapping:
 
 @
-"thread-id" -> myThreadId
-"utc-time"  -> getCurrentTime
+"threadId" -> myThreadId
+"utcTime"  -> getCurrentTime
 @
 -}
 defaultMessageMap :: MonadIO m => FieldMap m
 defaultMessageMap = fromList
-    [ TM.WrapTypeable $ MessageField @"thread-id" (liftIO myThreadId)
-    , TM.WrapTypeable $ MessageField @"utc-time" (liftIO getCurrentTime)
+    [ #threadId (liftIO myThreadId)
+    , #utcTime (liftIO getCurrentTime)
```

Comment:
This is the main motivating change in this PR

Reply:
Wow! That's magic!